src/main.ts

```diff
@@ -24,7 +24,7 @@ export async function main(): Promise<Outputs> {
     const managementGroupId = getInput('managementGroupId')
 
     // change the subscription context
-    if (scope != "managementgroup") {
+    if (subscriptionId !== "") {
```

Comment:
I think we should still have the check for `scope` as subscription should get ignored for managementGroup scope

Reply:
Ok, e08dd59 should fix this.